charts/localstack/templates/pvc.yaml

```diff
@@ -1,11 +1,15 @@
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+charts/localstack/templates/pvc.yaml{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: {{ include "common.names.fullname" . }}
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "localstack.labels" . | nindent 4 }}
+  {{- with .Values.pvcAnnotations }}
```

Comment:
Rather than doing a specific set of annotations for each resource type, I think we should create a top level `.Values.annotations` parameter and we should use that in all of the resources being  created.